.github/workflows/knative-releasability.yaml

```
@@ -40,35 +42,65 @@ jobs:
           go-version: 1.15.x
 
       - name: Install Dependencies
-        run: |
-          GO111MODULE=on go get tableflip.dev/buoy@master
+        run: GO111MODULE=on go get tableflip.dev/buoy@master
 
       - name: Check out code
         uses: actions/checkout@v2
+        with:
+          path: ./src/knative.dev/${{ github.event.repository.name }}
 
       - name: Check
         run: |
-          buoy check go.mod --release $RELEASE --verbose
+          echo 'CHECK_MESSAGE<<EOF' >> $GITHUB_ENV
+          (buoy check go.mod --release ${RELEASE} --domain knative.dev --verbose || CHECK=$1) >> $GITHUB_ENV
+          echo 'EOF' >> $GITHUB_ENV
```

Not sure if this is more or less confusing:
```suggestion
          cat >> $GITHUB_ENV <<END
          CHECK_MESSAGE<<EOF
          $(buoy check go.mod --release ${RELEASE} --domain knative.dev --verbose || CHECK=$?)
          EOF
          END
```

Either way `$1` is wrong 😉

---

my bash-fu is not good enough to understand things that use `>> sadsa <<`

My bad, I ment `$?`